### PR TITLE
Rebase s

### LINKED ISSUE #304: rebase (final?)

### PR DESCRIPTION
**Pull request by damianodamiani**
> Pull-requests should be ready to merge, that is free of conflicts, and they should pass our testsuite. If you require a specific reviewer you can request a review on the right.

### Please indicate issues related to this PR (if any)
See #253.
Close #304.

### Checklist
-   The title line explains what these changes do
-   All my changes are marked in the `last_modfied` column
-   The PR has no merge conflicts
-   Travis checks are passing

### CODE REVIEW

File: data/Act.csv [hunk 1]
```diff
@@ -572,7 +612,7 @@ A00817,P0020,AT01,PS09999,,1970..197x,1970..197x,The books were from the Nanjing
 A00818,P0020,AT01,PS09999,,1970..197x,1970..197x,The books were from the Nanjing secondary schools and were stored in an office after the schools were moved to the countryside,SS00006,134,2,unknown work by: P0999 G02 about P0232,,2018-10-24,MG,2018-10-24,MG
 A00819,P0020,AT01,PS09999,,1970..197x,1970..197x,The books were from the Nanjing secondary schools and were stored in an office after the schools were moved to the countryside,SS00006,134,2,unknown work by: P0999 G02 about P0651,,2018-10-24,MG,2018-10-24,MG
 A00820,P0020,AT01,PS09999,,1970..197x,1970..197x,The books were from the Nanjing secondary schools and were stored in an office after the schools were moved to the countryside,SS00006,134,2,unknown work by: P0999 G02 about P0652,,2018-10-24,MG,2018-10-24,MG
-A00821,P0120,AT23,PS00490,,1966..197x,1966..197x,oral transmission,SS00030,30; 152-153; 220; 251,2,,,2018-10-24,MG,2018-10-24,MG
```
Comment (duncdrum) [inline data/Act.csv]: removed on purpose?

File: data/Person.csv [hunk 2]
```diff
@@ -809,6 +809,47 @@ P0419,尼采,弗里德里希,zh,male,NULL,NULL,NULL,NULL,1844,1900,NULL,NULL,PL0
...
+P0434,Gao ,Mobo,en,male,unknown,unknown,unknown,unknown,1952,,Gao Changfang,en,PL0310,S03,unknown,https://en.wikipedia.org/wiki/Gao_Mobo,ONLINE SOURCE,,,,,2018-08-06,DM,2018-08-06,DM
+P0435,石,涛,zh,male,NULL,NULL,NULL,NULL,1642,1707,NULL,NULL,PL0311,NULL,NULL,https://en.wikipedia.org/wiki/Shitao,ONLINE SOURCE,,,,,2018-08-06,DM,2018-08-06,DM
+P0435,Shi,Tao,en,male,NULL,NULL,NULL,NULL,1642,1707,NULL,NULL,PL0311,NULL,NULL,https://zh.wikipedia.org/wiki/%E7%9F%B3%E6%B6%9B,ONLINE SOURCE,,,,,2018-08-06,DM,2018-08-06,DM
+P0436,Shirer,William L.,en,male,NULL,NULL,NULL,NULL,1904,1993,NULL,NULL,PL0312,NULL,NULL,https://en.wikipedia.org/wiki/William_L._Shirer?oldformat=true,ONLINE SOURCE,,,,,2018-08-06,DM,2018-08-06,DM
+P0436,夏伊勒,威廉·劳伦斯,zh,male,NULL,NULL,NULL,NULL,1904,1993,NULL,NULL,PL0312,NULL,NULL,https://zh.wikipedia.org/wiki/%E5%A8%81%E5%BB%89%C2%B7%E5%8B%9E%E5%80%AB%E6%96%AF%C2%B7%E5%A4%8F%E4%BC%8A%E5%8B%92?oldformat=true,ONLINE SOURCE,,,,,2018-08-06,DM,2018-08-06,DM
+P0437,高,瑞泉,zh,male,unknown,unknown,unknown,unknown,1948,,NULL,NULL,unknown,S03,unknown,https://baike.baidu.com/item/%E9%AB%98%E7%91%9E%E6%B3%89,ONLINE SOURCE,,,,,2018-08-06,DM,2018-08-06,DM
```
Comment (duncdrum) [inline data/Person.csv]: please use 'PL0999' for unknown places

File: data/Place.csv [hunk 3]
```diff
@@ -300,19 +313,19 @@ PL0357,Nanchang,28.682892,115.858197,,2018-07-10,WH,2018-07-10,WH
...
-PL0415,Hubei,37.598570,114.607580,,2018-11-09,MG,2018-11-09,MG
-PL0416,Warszawa ,52.229675,21.012230,,2018-11-09,MG,2018-11-09,MG
-PL0417,Pescasseroli,41.808250,13.789580,,2018-11-09,MG,2018-11-09,MG
-PL0999,unknown,0.00000,0.00000,,2018-11-13,MG,2018-11-13,MG
+PL0413,Caprese,43.64096,11.98596,,2018-11-06,MG,2018-11-06,MG
+PL0414,Alloway,55.43109,-4.6331,,2018-11-06,MG,2018-11-06,MG
+PL0415,Hubei,37.59857,114.60758,,2018-11-09,MG,2018-11-09,MG
+PL0416,Warszawa ,52.229675,21.01223,,2018-11-09,MG,2018-11-09,MG
+PL0417,Pescasseroli,41.80825,13.78958,,2018-11-09,MG,2018-11-09,MG
+PL0999,unknown,0,0,,2018-11-13,MG,2018-11-13,MG
```
Comment (duncdrum) [inline data/Place.csv]: this shouldn't be a problem but i would disable auto-formatting here as well in LO

File: data/Act.csv (hunk 1 above)
Comment (damianodamiani) [reply data/Act.csv]: no, didn't want to remove it...

File: data/Person.csv (hunk 2 above)
Comment (damianodamiani) [reply data/Person.csv]: ok, doing it now and pushing

File: data/Act.csv (hunk 1 above)
Comment (duncdrum) [reply data/Act.csv]: in which case we should put the page ranges back in `30; 152-153; 220; 251`

**Pull request merged**